docs/pages/sidenavigation.js

```diff
@@ -69,49 +56,342 @@ export default function SideNavigationPage({
+    <Heading accessibilityLevel="none" size={500}>Tag manager</Heading>
+  </Box>
+  <Flex>
+    <SideNavigation accessibilityLabel="Correct headings example" showBorder>
+      <SideNavigation.TopItem
+        href="#"
+        onClick={({ event }) => event.preventDefault()}
+        label="Tag manager"
+        icon="margins-small"
+        selected="section"
+      />
+      <SideNavigation.TopItem
+        href="#"
+        icon="refresh"
+        onClick={({ event }) => event.preventDefault()}
+        label="Upload file"
+      />
+      <SideNavigation.TopItem
+        href="#"
+        icon="visit"
+        onClick={({ event }) => event.preventDefault()}
+        label="Upload history"
+      />
+    </SideNavigation>
+    <Box padding={4}>
+      <Heading accessibilityLevel={2} size={400}>Tag manager</Heading>
+    </Box>
+  </Flex>
+</Flex>
+`}
+          />
+          <MainSection.Card
+            cardSize="lg"
+            type="don't"
+            description="Omit a PageHeader and make it seem like each SideNav item is a primary, independent page."
+            defaultCode={`
+<Flex width="100%">
+  <SideNavigation accessibilityLabel="Correct headings example" showBorder>
+    <SideNavigation.TopItem
+      href="#"
+      onClick={({ event }) => event.preventDefault()}
+      label="Tag manager"
+      icon="margins-small"
+      selected="section"
+    />
+    <SideNavigation.TopItem
+      href="#"
+      icon="refresh"
+      onClick={({ event }) => event.preventDefault()}
+      label="Upload file"
+    />
+    <SideNavigation.TopItem
+      href="#"
+      icon="visit"
+      onClick={({ event }) => event.preventDefault()}
+      label="Upload history"
+    />
+  </SideNavigation>
+  <Box paddingX={4}>
+    <Heading accessibilityLevel="none" size={500}>Tag manager</Heading>
+  </Box>
+</Flex>
+`}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
       <AccessibilitySection name={generatedDocGen.SideNavigation?.displayName}>
         <MainSection.Subsection
           title="Active item"
-          description='SideNavigation.TopItem has an "active" state that visually identifies it. To set SideNavigation.TopItem to active pass "page" (page redirect) or "section" (selected content section). Use routing hooks from React.Router or other frameworks to identify the current route. For example, if the current pathname matches the SideNavigation.TopItem href, set SideNavigation.TopItem to "page".'
+          description={`SideNavigation.TopItem has an "active" state that visually identifies it. To set SideNavigation.TopItem to "active" set 'active="page"' (page redirect) or 'active="section"'. Use routing hooks from React.Router or other frameworks to identify the current route. For example, if the current pathname matches the SideNavigation.TopItem href, set SideNavigation.TopItem to "page". use example below as a reference.`}
```

Comment:
typo at the end: Capitalize Use and add the: "Use the example..."

Reply:
DONE